filters/encrypt/README.md

```diff
@@ -0,0 +1,87 @@
+# encrypt package
+
+The encrypt package implements a new Filter that supports filtering fields in an
+event payload using a custom tag named `classification`.
```

Comment:
Tag name doesn't match here either.

Reply:
ty!

Reply:
This is marked as resolved but wasn't changed

Reply:
sorry about that.  I've pushed the commit now.